CHANGES.txt

```diff
@@ -36,6 +36,8 @@ Changes
 Fixes
 =====
 
+ - Return proper exception when group by is used on ``count(distinct)``.
```

Comment:
I think this is not count(distinct) specific but was the case for any scalar functions applied to an aggregation. E.g. `sum(x) + 1 + 2`